theme/templates/resource-landing-page/resource-header.html

```diff
@@ -92,9 +89,9 @@
                         This resource does not have an owner who is an active HydroShare user.
                         Contact CUAHSI (<a href="mailto:help@cuahsi.org">help@cuahsi.org</a>)
                         {% if cm.raccess.public %}
-                            for information on this resource.
+                        for information on this resource.
                         {% else %}
-                            to determine if accessing this resource is possible.
+                        to determine if accessing this resource is possible.
```

Comment:
all of your changes lines 11-94 appear to be whitespace/indentation changes.
Was this intentional?